Libraries/Components/ViewPager/ViewPagerAndroid.android.js

```diff
@@ -19,6 +19,12 @@ var requireNativeComponent = require('requireNativeComponent');
 
 var VIEWPAGER_REF = 'viewPager';
 
+var VIEWPAGER_PAGE_SCROLL_STATES = [
```

Comment:
I'd prefer to either send page state as a string from native, or expose mapping from state represented as an integer to the name of the state through ViewPagerManager constants (see [getExportedViewConstants](https://github.com/facebook/react-native/blob/dad462258fae332f276d9a256892cc5fe9fe0c14/ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewManager.java#L194) method).

Now this piece of code is coupled with the state values defined as constants in the android framework [here](http://developer.android.com/reference/android/support/v4/view/ViewPager.html#SCROLL_STATE_IDLE), and I'd really prefer to avoid this kind of coupling.